src/routes/safe/components/Transactions/helpers/EditTxParametersForm/index.tsx

```diff
@@ -132,7 +128,7 @@ export const EditTxParametersForm = ({
       <Block className={classes.container}>
         <GnoForm
           initialValues={{
-            safeNonce: safeNonce || recommendedNonce || '0',
```

Comment:
I removed the `recommendedNonce` here as it is the same fetch we already do for `safeNonce` so a fallback is unnecessary.